README.md

```diff
@@ -115,7 +115,8 @@ Install one or both of the following modules:
 Spawn the daemon
 
 - `options` is an optional object the following properties:
-  - `init` bool (default true) - should the node be initialized
+  - `init` bool (default true) or Object - should the node be initialized
+    - if `init` is an Object, it is expected to be of the form `{keySize: <bits>}`, this will set the `keysize` to initialize the daemon with
```

Comment:
Not a fan of conditional types in a parameter. Either it's a `Boolean` or it's a `Object`. Rather, we should add a new option. If we're only adding `keySize`, why not make `keySize` a key of the `options` object?

Reply:
I also think we call the option `bits` here, OR, change the option in go-ipfs as well. It's `bits` there currently, but `keySize` is probably a more descriptive name.

Reply:
Thanks @VictorBjelkholm, I thought about that too and you're right, I'll add another options `initOpts` and leave `init` as a bool.

For the second comment, `keysize` is used because historically that is what `ipfsd-ctl` used and it's the same name we use when passing that flag to `init`, and yes I think `keysize` is more descriptive.

Reply:
read: https://github.com/ipfs/js-ipfsd-ctl/issues/188#issuecomment-366299208

I'm fine with init being boolean or object, but I'm fine with initOpts too.

Reply:
README needs to be updated to include `initOptions`